docs/adrs/0004-agpl-v3-for-all-uds-products.md

```diff
@@ -0,0 +1,51 @@
+# 4. [Affero General Public License v3 (AGPLv3)](https://opensource.org/license/agpl-v3) for All UDS Products
```

Comment:
Do we need the title to reflect the dual license? I don't have a strong leaning either way, just wanted to ask.

Reply:
added that for clarity